front/src/utils/hooks/useModalFocusTrap.ts

```diff
@@ -14,10 +14,10 @@ export default function useModalFocusTrap(
     const focusableElements = modalElement?.querySelectorAll(
       // last declaration stands for all elements not natively focusable like li
       'input, button, [tabindex]:not([tabindex="-1"])'
-    ) as NodeListOf<Element>;
+    );
 
-    const firstElement = focusableElements[0] as HTMLElement;
-    const lastElement = focusableElements[focusableElements?.length - 1] as HTMLElement;
```

Comment:
As I now add the `?`, it was to avoid writing `HTMLElement | undefined`. But maybe I can it for a `!` instead of the cast as recommanded by @emersion.

Reply:
As @Yohh asked for it too, I put the cast back at the top.

Reply:
Ok new change. As eslint was warning for the cast, I added the `HTMLElement` directly on `querySelectorAll`.